documentation/modules/auxiliary/scanner/vnc/vnc_none_auth.md

```diff
@@ -0,0 +1,62 @@
+## Vulnerable Application
+
+Detect VNC servers that support the "None" authentication method.
+
+### Install
+
+TigerVNC is one of the VNC servers which still accepts None authentication type.
+
+#### Windows
+
+Follow https://github.com/TigerVNC/tigervnc/wiki/Setup-TigerVNC-server-(Windows) to download
+the server, and install with the default.  Next start `Configure VNC Service`.
+
+Set "Session encryption" to `None` and "Authentication" to `None`. Apply and restart the service.
+
+#### Linux
+
+tigervncserver is available on Ubuntu 18.04 and possibly newer versions. To start the server
+in a vulnerable way, run the following command:
+`tigervncserver -SecurityTypes None -localhost no --I-KNOW-THIS-IS-INSECURE`
```

Comment:
Should this also be the in `vnc_login.md` documentation as well since this seems applicable there?

Reply:
eh, debatable. I included it here since it is a specific config and flags that need to be implemented (notably, all 3 in combination). the normal `vnc_login` is just start the service and it works kinda thing.

Reply:
also of note, not all VNC servers still include a `None` type, and are available for free on linux, so documenting which one can be made to work in this case seemed important